dataland-dummy-qa-service/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG DATALAND_DUMMY_QA_SERVICE_BASE_VERSION
```

Comment:
We should name this thing QA_SERVICE, - only right now it will act as a dummy.

Reply:
Done

Reply:
ok